pycbc/population/rates_functions.py

```diff
@@ -23,8 +23,10 @@ def process_full_data(fname, rhomin, mass1, mass2, lo_tlt_mass, hi_tlt_mass):
               First mass of the waveform in the template bank
        mass2: array
               Second mass of the waveform in the template bank
-       min_bh_mass: float
-              Minimimum mass for the black hole
+       lo_mchirp: float
```

Comment:
Is it necessary to always give values for lo_mchirp and hi_mchirp?  What if we just wanted to accept all the triggers?  Maybe write the function in such a way that these arguments are optional?

Reply:
Recorded. Will change in future. For all the triggers I will just specify very small for low chirp mass and very high for hi_chirp mass.